packages/v4/src/content/developer-resources/patternfly-accessibility.md

```diff
@@ -0,0 +1,36 @@
+---
+id: PatternFly's Accessibility
+section: accessibility
+---
+
+As a design system, PatternFly provides accessible building blocks to work with. We do our best to cover all areas that are within our control and take at least some of the accessibility work off your hands.
+
+With constant enhancements to PatternFly, we maintain this accessibility through a combination of automated and manual testing. We use [aXe: The Accessibility Engine](https://www.deque.com/axe/) on our build to ensure that all components pass this accessibility audit before they’re added to PatternFly. 
+
+We regularly audit keyboard accessibility with both manual testing and integration tests. We’re also aiming to provide full support for Voice Over as our main screen reader, but we still test our components through NVDA and JAWS. As part of our manual audit, every component is run through Voice Over to make sure they’ll be as accessible as possible in your products.
+
+## What PatternFly should address
```

Comment:
Maybe this should say something more like "what PatternFly aims to address" so it doesn't sound like the read should do something?